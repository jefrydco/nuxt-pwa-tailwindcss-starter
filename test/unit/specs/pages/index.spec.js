import { shallowMount, createLocalVue, RouterLinkStub } from '@vue/test-utils'

import Index from '~/pages/index'
import { div } from '~/test/utils'

const localVue = createLocalVue()

describe('Index', () => {
  it('renders to match snapshot', () => {
    const wrapper = shallowMount(Index, {
      localVue,
      stubs: {
        'nuxt-link': RouterLinkStub,
        'nuxt-child': div
      }
    })
    expect(wrapper).toMatchSnapshot()
  })
})
