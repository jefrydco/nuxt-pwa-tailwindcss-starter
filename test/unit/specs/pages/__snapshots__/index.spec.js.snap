// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index renders to match snapshot 1`] = `
<div>
  <h1>Home</h1>
  <validation-provider-stub vid="" mode="aggressive" rules="numeric" bails="true" skipifempty="true" debounce="0" tag="span" custommessages="[object Object]"></validation-provider-stub> <a class="px-2 -mx-2 py-1 transition duration-200 ease-in-out relative block text-teal-600 font-medium underline">
    Inspire
  </a>
</div>
`;
